### DISCUSSION TITLE
Release V

### DISCUSSION THREAD

**Discussion by rosenangelov** [vanilophp/framework#112] [General]
Hi, i've searching for ecommerce laravel platform and i like very much your project, but checking last release and Roadmap i'm not sure if the project is still active. Regards!

**Comment by fulopattila122** [comment]
Hi @rosenangelov, that's a very valid question! :)

The project is alive, and 2.2 should've been released yet, but we committed the mistake of overstretching the scope of the release.
It will be out by next week and the roadmap will be updated.

Sorry for the confusion!

**Reply by rosenangelov** [reply]
Thank you @fulopattila122, i see that you update the Roadmap and this remove the feeling of an abandoned project, it's great to know that this project continues.

**Reply by fulopattila122** [reply]
Vanilo has been my primary gig since last fall, so definitely not abandoned.
I've already started the actual coding of 3.0 yesterday

**Reply by fulopattila122** [reply]
But our primary focus for the next 2 weeks is to finalize the PayPal, Stripe and Adyen plugins. They are all in a working condition, but aren't polished yet and their documentation is also noisy.

Which one of those 3 (paypal, adyen, stripe) would you be your primary go-to payment solution?
